scripts/start.js

```diff
@@ -72,6 +73,20 @@ function clearConsole() {
   process.stdout.write('\x1bc');
 }
 
+function getExistingProcesses(port) {
+  //determine if another process running on the desired port
+  return new Promise(function (resolve, reject) {
+    var command = 'lsof -P | grep TCP | grep :' + port + ' | cut -d \' \' -f 1';
```

Comment:
Can you also run the second line I mentioned this way you get the entire command that started it instead of just "node". This way we get the entire path of the app running which should be more helpful :)